lib/db_connection/ownership/manager.ex

```diff
@@ -98,6 +104,38 @@ defmodule DBConnection.Ownership.Manager do
   end
 
   @impl true
+  def handle_call(:get_connection_metrics, _from, %{pool: pool, owners: owners, log: log} = state) do
+    {:ok, pool_metrics} = DBConnection.ConnectionPool.get_connection_metrics(pool)
+
+    proxy_metrics =
+      owners
+      |> Enum.map(fn {_, {proxy, _, _}} ->
+        if Process.alive?(proxy) do
+          GenServer.call(proxy, :get_connection_metrics)
+        end
+      end)
+      |> Enum.reject(&is_nil/1)
+
+    {:reply, {:ok, pool_metrics ++ proxy_metrics}, state}
+  rescue
+    error in MatchError ->
```

Comment:
When would this happen?

Reply:
It literally wouldn't. It was meant to hightlight the return type of `ConnectionPool.get_connection_metrics`

Reply:
So we can remove this then :)

Reply:
Deleted it since it causes confusion